src/util/delay.test.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-env jest */
+
+import delay from './delay'
+jest.useFakeTimers()
+
+describe('delay', () => {
+    test('should return with the given delay', () => {
+        delay(1000)
+        expect(setTimeout.mock.calls.length).toBe(1)
+        expect(setTimeout.mock.calls[0][1]).toBe(1000)
+    })
+
+    test('should return a promise which resolves', () => {
```

Comment:
Please make this test more straight forward. expect directly on the returned promise of delay(10).